src/.bazelrc

```diff
@@ -19,8 +16,12 @@ build --remote_upload_local_results=false
 # the "factory" design pattern.
 build --incompatible_remove_legacy_whole_archive=False
 
-# Every warning is treated as an error
-build --cxxopt=-Werror
-
```

Comment:
Why are we deleting this?

Reply:
All build flags are part of the `cc_toolchain` now, in `cc_toolchain_config.bzl `.

Reply:
All build flags are part of the `cc_toolchain` now, in `cc_toolchain_config.bzl `.